include/seqan/platform/platform_windows.h

```diff
@@ -71,6 +71,11 @@
 #pragma warning( disable : 4345 )
 #pragma warning( disable : 4351 )
 
+// allow multiple constructors
+#pragma warning( disable : 4521 )
```

Comment:
ok done, I also removed the #define _CRT_SECURE_NO_WARNINGS. 
I noticed that most of my warnings appear when i build all the demos. The demos are not built on travis. Why are they excluded there?